### PR TITLE
added profile component to seller dashboard and a form to edit their information 

### PR DESCRIPTION
**Pull request by aland1013**
## MOTIVATION
to allow sellers to view and edit their profiles
## WHAT YOU DID
added a SellerInfo component to the seller dashboard and a modal that has a form to edit their information
## TRELLO CARD
image
## USER STORY
as a seller, I want to be able to view and update my profile information
## TESTS
the components were tested locally
## BUGS
no bugs found

### CODE REVIEW

**Comment by aland1013** [comment]
[(?) As a seller, I would like to see and edit my seller profile](https://trello.com/c/MCkeQCKH/4-as-a-seller-i-would-like-to-see-and-edit-my-seller-profile)

**Review by christian-mack — DISMISSED** [review]
Great. Looks like this is ready to be styled

File: src/components/sellerPages/profile/dashboard/infoSection/EditProfileForm.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+  useEffect(() => {
+    setAddress(fields.length > 0 ? fields[2].value : '');
+    setAddressValue(fields.length > 0 ? fields[2].value : '');
+  }, [fields]);
+
+  const [form] = Form.useForm();
+
+  const validatePhone = (rule, value, callback) => {
+    const regex = new RegExp(/\([0-9]{3}\)\s[0-9]{3}-[0-9]{4}/);
```
Comment (BkAngel201) [inline src/components/sellerPages/profile/dashboard/infoSection/EditProfileForm.js:18]: this phoner validation is really nice, that way we keep consistency in the data submitted 😃

File: src/components/sellerPages/profile/dashboard/infoSection/EditProfileForm.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            {
+              required: true,
+              message: 'Please enter a description',
+            },
+          ]}
+        >
+          <Input.TextArea />
+        </Form.Item>
+        <Text>Address: </Text>
+        <AutoComplete
```
Comment (BkAngel201) [inline src/components/sellerPages/profile/dashboard/infoSection/EditProfileForm.js:80]: Google Map API, yeah I like it, this component is Awesome

**Review by BkAngel201 — DISMISSED** [review]
All looks good and work fine just a little bit of style at the end and solid seller dashboard

**Review by christian-mack — DISMISSED** [review]
These changes look good

**Review by BkAngel201 — DISMISSED** [review]

**Comment by aland1013** [comment]
I cleared a few merge conflicts

**Review by BkAngel201 — APPROVED** [review]

File: src/components/sellerPages/profile/dashboard/invSection/index.js [hunk 3]
```diff
@@ -12,8 +12,6 @@ function InvSection({ inventory, fetchProducts, getProductsStatus }) {
     fetchProducts(authState);
   }, []);
 
-  console.log(inventory);
```
Comment (christian-mack) [inline src/components/sellerPages/profile/dashboard/invSection/index.js:15]: Good catch on removing this

**Review by christian-mack — APPROVED** [review]

**Pull request merged**